data/assets/scene/solarsystem/missions/jwst/timelapse.asset

```diff
@@ -520,6 +524,8 @@ local function createActions()
       openspace.scriptScheduler.clear(0)
       openspace.time.setDeltaTime(1)
       openspace.setPropertyValueSingle("Dashboard.JWSTStateText.Text", "")
+      openspace.setPropertyValueSingle("Scene.Earth.Renderable.Layers.NightLayers.Earth_at_Night_2012.Settings.Gamma", 1.0)
```

Comment:
Are we less strict with max column with for these kinds of scripts? (looks like it from he rest of this file :) )

And I can agree that it's probably more readable to have it in one line

Reply:
Since it is a Lua script in string form it would be tricky to enforce the column width style here I think